docs/DOCS_README.md

```diff
@@ -2,39 +2,38 @@
 
 The documentation for Tendermint Core is hosted at:
 
-- <https://docs.tendermint.com/master/>
+- <https://docs.tendermint.com>
 
-built from the files in this (`/docs`) directory for
-[master](https://github.com/tendermint/tendermint/tree/master/docs) respectively.
+built from the files in this (`/docs`) directory.
 
 ## How It Works
 
-There is a CircleCI job listening for changes in the `/docs` directory, on both
-the `master` branch. Any updates to files in this directory
-on those branches will automatically trigger a website deployment. Under the hood,
-the private website repository has a `make build-docs` target consumed by a CircleCI job in that repo.
+There is a [GitHub Action](../.github/workflows/docs-deployment.yml) that is
```

Comment:
I believe if this is rendered in the docs site it will be wrong, but it's fine if that's never the case. It's always safe to do absolute/full URL links

Reply:
It is [rendered](https://docs.tendermint.com/master/DOCS_README.html), but it's not actively linked to from anywhere in the docs. The GitHub version of this doc is linked to from [here](https://docs.tendermint.com/master/#contribute).